templates/env.j2

```diff
@@ -1,4 +1,6 @@
 WOODPECKER_SERVER={{ devture_woodpecker_ci_agent_config_server }}
+WOODPECKER_GRPC_SECURE={{ devture_woodpecker_ci_agent_config_server_use_tls | to_json }}
+WOODPECKER_GRPC_VERIFY={{ devture_woodpecker_ci_agent_config_server_verify_tls | to_json }}
```

Comment:
All Ansible variables are named in a consistent way that hints as to what they control.

A variable `devture_woodpecker_ci_agent_config_server` includes:

- the `devture_woodpecker_ci_agent` prefix for the role
- a `config` part that tells us it's related to configuration
- a `server` part which tells us it configures some `SERVER` variable (`WOODPECKER_SERVER`)

----

Due to this, your Ansible variables should be named `devture_woodpecker_ci_agent_config_grpc_secure` and `devture_woodpecker_ci_agent_config_grpc_verify`.

I agree that `use_tls` and `verify_tls` explain the intention better, but they're deviating from our naming conventions, which can be confusing in its own way.

Reply:
Ah, thanks for the review.  Fixed in the last force-push.